specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/examples/GetVirtualWanVpnServerConfigurations.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "api-version": "2019-08-01",
+    "virtualWANName": "wan1"
+  },
+  "responses": {
```

Comment:
Specs also define `202` response, please add it to example (it should be empty though)

Reply:
Done.